tfx/components/example_validator/executor_test.py

```diff
@@ -29,45 +29,72 @@
 
 class ExecutorTest(tf.test.TestCase):
 
-  def testDo(self):
```

Comment:
Can you make sure this works with and without the optional argument?

Reply:
I added testDoSkewDetection() which uses the optional argument, and the existing testDo() does not use it, both versions seem to work. I also ended up extracting some common code into a setUp() and a helper method.